### DISCUSSION TITLE
How to delivery hard fails back to sender

### DISCUSSION THREAD

**Discussion by udamaster** [postalserver/postal#2483] [Help with using Postal]
Coming from postfix world ...we've moved over one of our distro lists and I'm trying to get the bounce management running smoothly. Seems like Postal will just add addresses into the suppression list but hard fails aren't returning to sender? I must be missing something obvious.

We want to ensure subscribers are unsubscribed ...is web hook impl the only way to do this? Can we direct hard fails to the return to sender so existing process can pickup from bounce email account?

Appreciate any insight, thanks!

**Comment by willpower232** [comment]
Postal is designed for software to send messages so doesn't send traditional bounceback messages. You can make use of webhooks to listen to events when messages change status.
